page/Views/layout.php

```diff
@@ -0,0 +1,31 @@
+                <span class="fs-4">Translation Status</span>
+            </a>
+        </header>
+
+        <?= $this->renderSection('content') ?>
+
+        <footer class="pt-3 mt-4 text-muted border-top">
+            &copy; <?= date('Y'); ?> CodeIgniter Foundation. CodeIgniter is open source project released under the MIT open source licence.
```

Comment:
Should the copyright date be fixed?

Reply:
Honestly, I have no idea. I thought it would be a good idea to have a year associated with the moment the site was generated.

Reply:
I don't think it matters for anything legal. These website copyright tags are basically just FYIs. I think fixed dates are helpful as a sort of "page last updated" but that's also a case for using the current date, so it doesn't have to be touched all the time.

Reply:
I wanted it to be an indicator of the last build. A fixed date is one more thing we have to remember about.

If anyone feels this should be changed, please add a code suggestion, and I will accept it.

Reply:
I put a suggestion to align with the license.
```suggestion
            &copy; 2019-<?= date('Y'); ?> CodeIgniter Foundation. CodeIgniter is open source project released under the MIT open source licence.
```

Reply:
Thanks!